fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/ServerConfigurationNetworking.java

```diff
@@ -235,6 +237,26 @@ public static MinecraftServer getServer(ServerConfigurationNetworkHandler handle
 		return ((ServerCommonNetworkHandlerAccessor) handler).getServer();
 	}
 
+	/**
+	 * Sets the cookie data on the client.
+	 *
+	 * @param cookieId The id to tag the data with.
+	 * @param cookie The data to be set on the client.
+	 */
+	public static void setCookie(ServerConfigurationNetworkHandler handler, Identifier cookieId, byte[] cookie) {
+		((ServerCookieStore) handler).setCookie(cookieId, cookie);
+	}
+
+	/**
+	 * Retrieves cookie data from the client.
+	 *
+	 * @param cookieId The id the data was tagged with.
+	 * @return The cookie data or an empty byte[] if there was no cookie found with that id.
+	 */
+	public static CompletableFuture<byte[]> getCookie(ServerConfigurationNetworkHandler handler, Identifier cookieId) {
+		return ((ServerCookieStore) handler).getCookie(cookieId);
```

Comment:
This should be something along the lines of:

```suggestion
		return ServerNetworkingImpl.getAddon(handler).getCookie(cookieId);
```

Reply:
Done